core-methods/src/main/java/com/mercadopago/sdk/android/coremethods/ui/components/textfield/expirationdate/ExpirationDateTextField.kt

```diff
@@ -0,0 +1,102 @@
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+import com.mercadopago.sdk.android.coremethods.ui.components.PreviewGroup
+import com.mercadopago.sdk.android.coremethods.ui.components.textfield.ExpirationDateLength
+import com.mercadopago.sdk.android.coremethods.ui.components.textfield.pcitextfield.PCIFieldState
+import com.mercadopago.sdk.android.coremethods.ui.components.textfield.pcitextfield.PCITextField
+import com.mercadopago.sdk.android.coremethods.ui.components.textfield.pcitextfield.PCITextFieldTestTags
+import com.mercadopago.sdk.android.coremethods.ui.components.textfield.pcitextfield.rememberPCIFieldState
+import com.mercadopago.sdk.android.coremethods.ui.components.textfield.securitycode.SecurityCodeTextField
+
+/**
+ * Expiration date input component.
+ *
+ * This component allows users to enter a card expiration date.
+ * It integrates the [PCIFieldState] that manages the entry and provides information of state of the field.
+ *
+ * @param modifier Modifier to customize the style and behavior of the field.
+ * @param state A [PCIFieldState] object that contains and manages the input data for the security field.
+ * @param onEvent A callback triggered in response to field events, such as focus changes or value changes.
+ * @param enabled Controls the enabled state of the [SecurityCodeTextField], allowing or preventing user interaction.
+ * @param readOnly Controls whether the field is editable or read-only.
+ * @param decorationBox A composable that allows the addition of decorative elements around the text field.
+ * @param textStyle Text style to be applied to the field's content.
+ * @param cursorBrush Brush applied to the text field's cursor, allowing customization of the cursor's appearance.
+ */
+@Composable
+fun ExpirationDateField(
+    modifier: Modifier = Modifier,
+    state: PCIFieldState,
+    onEvent: (ExpirationDateFieldEvent) -> Unit,
+    enabled: Boolean = true,
+    readOnly: Boolean = false,
+    decorationBox: @Composable (
+        innerTextField: @Composable () -> Unit
+    ) -> Unit = @Composable { innerTextField -> innerTextField() },
+    textStyle: TextStyle = MaterialTheme.typography.bodyLarge,
+    cursorBrush: Brush = SolidColor(MaterialTheme.colorScheme.primary),
+) {
+    PCITextField(
+        value = state.input,
+        onFocusChanged = { isFocused ->
+            onEvent(ExpirationDateFieldEvent.OnFocusChanged(isFocused))
+        },
+        onValueChange = { value ->
+            if (value.length <= ExpirationDateLength && value.none { !it.isDigit() }) {
+                onEvent(ExpirationDateFieldEvent.OnInputFilled(isFilled = value.length == ExpirationDateLength))
+                onEvent(ExpirationDateFieldEvent.OnLengthChanged(length = value.length))
+                state.input = value
+            }
+        },
+        keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
```

Comment:
```suggestion
           keyboardOptions = keyboardOptions.copy(keyboardType = KeyboardType.Number),
```

we could do this to allow them to change the IME actions but still force the keyboard to be numbers